projects/plugins/jetpack/modules/protect/transient-cleanup.php

```diff
@@ -1,30 +1,35 @@
 <?php
-/*
-Adapted from Purge Transients by Seebz
-https://github.com/Seebz/Snippets/tree/master/Wordpress/plugins/purge-transients
-*/
+/**
+ * Adapted from Purge Transients by Seebz
+ * https://github.com/Seebz/Snippets/tree/master/Wordpress/plugins/purge-transients
+ *
+ * @package automattic/jetpack
+ */
+
 if ( ! function_exists( 'jp_purge_transients' ) ) {
 
 	/**
 	 * Jetpack Purge Transients.
 	 *
 	 * @access public
 	 * @param string $older_than (default: '1 hour') Older Than.
-	 * @return void
 	 */
 	function jp_purge_transients( $older_than = '1 hour' ) {
 		global $wpdb;
 		$older_than_time = strtotime( '-' . $older_than );
 		if ( $older_than_time > time() || $older_than_time < 1 ) {
 			return false;
 		}
-		$sql = $wpdb->prepare( "
+		$sql           = $wpdb->prepare(
+			"
 		SELECT REPLACE(option_name, '_transient_timeout_jpp_', '') AS transient_name
 		FROM {$wpdb->options}
 		WHERE option_name LIKE '\_transient\_timeout\_jpp\__%%'
```

Comment:
You still have some PHPCS failures in here.

Reply:
Yeah I'm not sure about these. I think for that first one, we can ignore it? As far as I can see `\_transient\_timeout\_jpp\__%%` is already using a replacement parameter `$older_than_time` right? 

For the second one, `Use placeholders and $wpdb->prepare(); found $sql
(WordPress.DB.PreparedSQL.NotPrepared)` - $sql is already `prepared` no? Can that one be ignored as well?

Just being very cautious with db stuff.